Wrappers/Python/ccpi/viewer/CILViewer2D.py

```diff
@@ -76,10 +76,10 @@ def isAllInactive(self):
         return all(not x for x in self.events.values())
 
 
-class CILInteractorStyle(vtk.vtkInteractorStyleImage):
+class CILInteractorStyle(vtk.vtkInteractorStyle): #vtk.vtkInteractorStyleImage
```

Comment:
Here I changed the interactor style we inherit from, in case it was the key press interactions from vtk.vtkInteractorStyleImage that was causing problems. Changing this made no difference.